### ISSUE TITLE
Include clade type in phyloreference definition

### ISSUE THREAD

**Issue by gaurav** [phyloref/phyx.js#6]
Each phyloref definition should include a clade type. To start with, we could include all the clade types currently supported by PhyloRegnum, which includes "apomorphy-based_standard", "apomorphy-modified_crown_clade", "crown-based_total_clade", "minimum-clade_standard", "maximum-crown-clade", "maximum-total-clade", "maximum-clade_standard" and "minimum-crown-clade" (I'll get a full list from Nico when I implement this). This information could then be incorporated into generated ontologies to distinguish between crown-clades and total-clades (phyloref/phyloref-ontology#33).

**Comment by gaurav** [comment]
Decision: let's make this a verbatim text field for now.

**Comment by gaurav** [comment]
After discussing this on the Phyloref call on January 27, 2021, we decided to have a `phylorefType` field that has one of the following values:
- `phyloref:PhyloreferenceUsingMinimumClade` for minimum-clade phylorefs.
- `phyloref:PhyloreferenceUsingMaximumClade` for maximum-clade phylorefs.
- `phyloref:PhyloreferenceUsingApomorphy` for apomorphy-based phylorefs.

These values would then be added to the list of superclasses for the phyloreference, so that a minimum-class phyloreference would be a subclass of phyloref:PhyloreferenceUsingMinimumClade. I've implemented this in PR #61.
